packages/api/src/controllers/rounds-controller.ts

```diff
@@ -169,17 +169,22 @@ type CompleteRound = Round & {
  * @returns The newly created round ID.
  */
 async function duplicateRound(originalRound: CompleteRound): Promise<number> {
-  return await prisma.$transaction(async (tx: Tx) => {
-    const newRound = await duplicateRoundRecord(tx, originalRound);
+  return await prisma.$transaction(
+    async (tx: Tx) => {
+      const newRound = await duplicateRoundRecord(tx, originalRound);
 
-    for (const category of originalRound.categories) {
-      const newCategory = await duplicateCategory(tx, category, newRound.id);
+      for (const category of originalRound.categories) {
+        const newCategory = await duplicateCategory(tx, category, newRound.id);
 
-      await duplicateSteps(tx, category.steps, newCategory);
-    }
+        await duplicateSteps(tx, category.steps, newCategory);
+      }
 
-    return newRound.id;
-  });
+      return newRound.id;
+    },
+    {
+      timeout: 30_000,
```

Comment:
@martinc1991, based on what I'm seeing in [the documentation](https://www.prisma.io/docs/orm/prisma-client/queries/transactions#transaction-options), the timeout is in milliseconds (30000 and not 30_000).

Reply:
in case we want, we could also change the timeout globally (all the trx) at the constructor. in this case -> https://github.com/wakeuplabs-io/optimism-making-impact/blob/develop/packages/api/src/lib/prisma-instance.ts